var/spack/repos/builtin/packages/pythia8/package.py

```diff
@@ -58,6 +58,14 @@ class Pythia8(AutotoolsPackage):
         deprecated=True,
     )
 
+    cxxstds = ("11", "17", "20", "23")
+    variant(
+        "cxxstd",
+        default="11",
+        values=cxxstds,
```

Comment:
Do you want to add `multi=False` here too?

Reply:
If `multi=False` is not the default, then yes, please add it!

Reply:
Added